CHANGELOG.md

```diff
@@ -46,6 +46,7 @@
 
 ## stream-chat-android-ui-components
 ### 🐞 Fixed
+- Fixed the links in UI Components code snippets. [#3261](https://github.com/GetStream/stream-chat-android/pull/3261)
```

Comment:
Should we mention this in the CHANGELOG? 🤔

Reply:
Let's discuss with the team, created a card for that on our Trello board.